src/store/app.js

```diff
@@ -34,6 +34,9 @@ const actions = {
         action: 'api/v1/notifications'
       })
       .then(response => {
+        if (response.headers.get('Content-Length') === '0') {
```

Comment:
```suggestion
        if (parseInt(response.headers.get('Content-Length')) === 0) {
```

Reply:
Why is that better?
The result would be the same but cost more because of the parsing.

Reply:
Yeah sorry... always a string anyway 🤦